src/signin/layout/index.tsx

```diff
@@ -0,0 +1,15 @@
+import S from "./Layout.module.scss";
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const Layout = ({ children }: LayoutProps) => {
```

Comment:
React에서 제공하는 PropsWithChildren이라는 타입을 활용해볼 수도 있습니다.
```suggestion
interface LayoutProps extends React.PropsWithChildren {}

const Layout = ({ children }: LayoutProps) => {
```